src/main/java/com/company/zayats/hw16/Dictionary.java

```diff
@@ -0,0 +1,33 @@
+package com.company.zayats.hw16;
+
+import java.util.*;
+
+public class Dictionary {
+
+    private Map<String, Integer> dictionary = new HashMap<>();
+
+    public Dictionary() {
+        dictionary.put("juice", 2);
+        dictionary.put("money", 5);
+        dictionary.put("car", 1);
+        dictionary.put("lion", 6);
+        dictionary.put("weather", 8);
+        System.out.println(dictionary);
+    }
```

Comment:
A string of text should be supplied to the input and a dictionary is built on its basis